go/serial/schema.fbs

```diff
@@ -22,6 +22,7 @@ table TableSchema {
   clustered_index:Index (required);
   secondary_indexes:[Index];
   checks:[CheckConstraint];
+  collation:uint16;
```

Comment:
In all other instances, SQL type information (including column collations) is stored as string data that can be interpreted by the SQL parser. We should either replicate that pattern here, or create a flatbuffers type alias for `uint16` and enumerate all possible values for `collation`. See `serial.Encoding` as an exampl

Reply:
I like the FlatBuffers type alias, I’ll implement that.